src/mailing/route.js

```diff
@@ -0,0 +1,11 @@
+import middyfy from 'middleware/wrapper';
+
+import { successCB } from "../callbacks/shared";
```

Comment:
nit: add a path alias to `webpack.config.js` for `src/mailing` so that we don't need relative imports